frontend/src/pages/ArtifactList.test.tsx

```diff
@@ -137,6 +163,26 @@ describe('ArtifactList', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  it('Renders and sorts Artifacts in ascending order by version',
```

Comment:
Possible to add test(s) for row expansion?

Reply:
Done. Tests added for a row with children and one without.